static/js/modules/tables.js

```diff
@@ -605,14 +610,28 @@ DataTable.prototype.fetchSuccess = function(resp) {
 DataTable.prototype.fetchError = function() {
   var self = this;
 
-  setTimeout(function() {
-    $('.is-loading').removeClass('is-loading').addClass('is-unsuccessful');
-    self.$processing.hide();
-  }, helpers.LOADING_DELAY);
+  $('.filter__message').remove();
+
+  $('.filters .is-loading')
+    .removeClass('is-loading')
+    .addClass('is-unsuccessful')
+    .after($('<div class="filter__message filter__message--error">' +
+      '<strong>We had trouble processing your request</strong><br>' +
+      'Please try again. If you still have trouble ' +
+      '<span class="js-filter-feedback">let us know</a></div>'))
```

Comment:
Inconsistent tag (`a` vs `span`). And I _think_ this should be a button, since it's just opening something and not taking you somewhere.